File: AceptaElReto/219-LoteriaAtletica.c

#include <stdio.h>

int main(int argc, char *argv[])
{

    int count, proc = 0;
    scanf("%d", &count);

    while (proc < count){

        int decimos, i, total = 0;
        scanf("%d", &decimos);

        for (i=0; i<decimos; i++){
            int value;
            scanf("%d", &value);
            if (value%2==0) total++;
        }

        printf("%d\n", total);

        proc++;

    }

    return 0;

}

File: AceptaElReto/219-LoteriaAtletica.java
import java.util.Scanner;

public class LoteriaAtletica {

    public static void main(String[] args) {
        
        Scanner s = new Scanner(System.in);
        int count = Integer.parseInt(s.nextLine());
        int proc = 0;

        while (proc < count){

            int decimos = Integer.parseInt(s.nextLine());
            int total = 0;

            for (int i=0; i<decimos; i++){
                if (s.nextInt()%2==0) total++;
            }
            s.nextLine();

            System.out.println(total);

            proc++;

        }
        
    }



    
}